### PR TITLE
refactor : image size and alignment  in readme

### PR DESCRIPTION
**Pull request by rtharungowda**
Center aligned and resized images to have almost the same height.
Fixes issue - README.md: Fix inconsistent image widths DevoLearn#12

### CODE REVIEW

**Comment by Mayukhdeb** [comment]
Hi  @rtharungowda, thanks for the PR. I'll review it soon :+1:

File: README.md [hunk 1]
```diff
@@ -26,15 +26,18 @@ pip install devolearn
 ```
 ### Example notebooks
 <code>
-<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/nodes_matrix_long_smooth.gif" width = "40%">
-<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/3d_node_map.gif" width = "30%">  
+<p align="center">
+<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/nodes_matrix_long_smooth.gif" width = "40%" height="200">
+<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/3d_node_map.gif" width = "40%" height="200">  
+</p>
 </code>
```
Comment (Mayukhdeb) [inline README.md:32]: Please remove the `height` argument, it breaks the aspect ratio of the GIF when resizing the window

Comment (rtharungowda) [reply README.md:32]: Ok on it!

Comment (Mayukhdeb) [reply README.md:32]: Please mention the issue on the commit  message as:

`readme: fix #30: align images`

**Pull request merged**